lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -1313,10 +1313,11 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
 
   def alert_ids(%__MODULE__{} = t), do: [t.alert.id]
 
-  def valid_candidate?(%__MODULE__{} = t) do
-    test_alert_ids = ["197140"]
-    prod_alert_ids = ["580015"]
-    t.alert.id not in (test_alert_ids ++ prod_alert_ids)
+  @test_alert_ids ~w[197140]
+  @prod_alert_ids ~w[580015 590467]
```

Comment:
Harmless to leave them, but the IDs that were already here can be removed. Alerts can't be reopened so we know those IDs won't ever be active again.

Reply:
[Done!](https://github.com/mbta/screens/compare/90b443cc218239221a03ab279021a00c63079094..9841db120da6be30070a3ce50ddac4c527c89a92) 🚮